libraries/mbed/targets/hal/TARGET_Freescale/TARGET_K20XX/TARGET_K20D50M/PeripheralPins.c

```diff
@@ -0,0 +1,120 @@
+
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
```

Comment:
Cosmetic change - Can you extend license for t hose files you changed to 2015 ? And files like for Teensy, should have probably only 2015 ?